examples/pyplots/pyplot_simple.py

```diff
@@ -3,8 +3,24 @@
 Pyplot Simple
 =============
 
+A most simple plot, where a list of numbers is plotted against their index.
 """
 import matplotlib.pyplot as plt
 plt.plot([1,2,3,4])
 plt.ylabel('some numbers')
 plt.show()
+
+#############################################################################
```

Comment:
At this point I'm wondering if there is not a way to automate this block of text that is cut and pasted between 52 files here.  If we decide the change the heading from "References" to "Methods in this example" we will have to go through all these changes and edit by hand.  I'm not familiar enough w/ Sphinx/gallery to know if there is a macro definition possible, but maybe we should check if there is?

Reply:
As far as I can tell there is no way to manipulate the way sphinx-gallery processes the `.py` files - other than monkey-patching sphinx-gallery. I took this approach in #11169. However since here we need to change the content of this "Reference" section manually, such approach would be really complicated. 

However, I think it should be relatively easy to change the section heading via search&replace (possibly with regex) in case that would be needed in the future.

Reply:
For the record I have a work in the plans to make this kind of referencing completely automated (using a rewrite from scratch of s-g, as there are a bunch of things I don't like in their design anyways).
(The idea being to inject an AST transformer to transform foo.bar to `logged_getattr(foo, "bar")` at the AST level, so that I know what is the actual name that needs to be linked at what point in the source.)

Reply:
@anntzer On what timescale do you plan to do this? In how far should this prevent us from continuing adding the backreferences manually?

Reply:
I "should" have something by the end of the month.